pkg/inject/utils.go

```diff
@@ -80,3 +80,33 @@ func containsEnvoyContainer(pod *corev1.Pod) (bool, int) {
 	}
 	return false, -1
 }
+
+func isSDSDisabled(pod *corev1.Pod) bool {
+	if v, ok := pod.ObjectMeta.Annotations[AppMeshSDSAnnotation]; ok {
+		if v == "disabled" {
+			return true
+		}
+	}
+	return false
+}
+
+func mutateSDSMounts(pod *corev1.Pod, envoyContainer *corev1.Container, SdsUdsPath string) {
+	SDSVolumeType := corev1.HostPathSocket
+	volume := corev1.Volume{
+		Name: "sds-socket-volume",
```

Comment:
nit: prefix the name with "appmesh" so the name is not super generic and can be identified for debugging etc

Reply:
Sure, will change it.